### PR TITLE
Improvements in multi-language, small improvements in layouts

### PR DESCRIPTION
**Pull request by domingoruiz**
Hello, I have made some improvements. First I have added a meta tag with all the languages ​​of the site, so that it can be recognized by the browser. In addition, I have added the language of the site in the <html> tag. Finally I have added comments in the head.html file.

Thanks

### CODE REVIEW

File: layouts/partials/head/head.html [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-
 {{ $styles = $styles | minify }}
 <link href="{{ $styles.Permalink }}" rel="stylesheet" />
-
 <link href="/fonts/rocky-font.css" rel="stylesheet" type="text/css" />
+
+<!-- Favicon -->
 <link rel="icon" href="/img/favicon.ico"/>
 
+<!-- Javascrip -->
```
Comment (FoggyMtnDrifter) [inline layouts/partials/head/head.html]: Minor change, but make sure this is spelled "JavaScript".

Comment (hbjydev) [reply layouts/partials/head/head.html]: (irrelevant to this comment, but @ressonix -- quick GitHub tip for you: if you make a codeblock with the type `suggestion`, you can get a diff-like comment, see below)
```suggestion
<!-- JavaScript -->
```

Comment (domingoruiz) [reply layouts/partials/head/head.html]: Sorry, typo problem. I solve it now. Thank you.

**Comment by domingoruiz** [comment]
Ok solve @ressonix 

Thanks

**Pull request merged**